tasks/queue-granules/index.js

```diff
@@ -76,78 +197,92 @@ async function queueGranules(event, testMocks = {}) {
   const enqueueGranuleIngestMessageFn
     = testMocks.enqueueGranuleIngestMessageMock || enqueueGranuleIngestMessage;
 
-  const collectionConfigStore = new CollectionConfigStore(
-    event.config.internalBucket,
-    event.config.stackName
+  const memoizedFetchProvider = memoize(fetchGranuleProvider, (_, providerId) => providerId);
+  const memoizedFetchCollection = memoize(
+    collectionsApi.getCollection,
+    ({ collectionName, collectionVersion }) => constructCollectionId(
```

Comment:
`getCollection` takes `prefix, collectionName, collectionVersion`.  I'm wondering how the integration-test passes.

Reply:
getCollection is called with the prefix on line 231. I think you're confusing the second arg as the call to the api method but it's actually the key for the memoize.